docker-compose.yml

```diff
@@ -30,6 +30,9 @@ services:
       DB_HOST: postgres
       DB_PORT: 5432
       START_BLOCK: 1
+      ## Set the below two values to run all migration required from version `PREVIOUS_SQ_VERSION` to `CURRENT_SQ_VERSION`
+      PREVIOUS_SQ_VERSION: 5.3.8
```

Comment:
I'd prefer if the user could just start the new version and have these be automatically ran with config to enable auto migration. (maybe default to auto migrate?). The user has already specified the versions so this seems like double input that will be error prone.

SQ has sequlize as a dependency, so we should be able to use that migration system: https://sequelize.org/docs/v6/other-topics/migrations/

The CLI should be scriptable and ran with the one making tooling-gql style tables.